### PR TITLE
add installation instructions for mac os

### PR DESCRIPTION
**Pull request by martenmatrix**
sadly project does not get built for arm architecture, idk if just commenting out arm64 in the gobuilder fixes this ¯\_(🥔)_/¯

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,6 +33,21 @@ sudo cp ./stine /usr/bin/stine
...
 
+### macOS
+Download and unpack the latest release:
+```bash
+# download
+curl -O -L https://github.com/irgendwr/go-stine/releases/latest/download/stine_Darwin_x86_64.tar.gz
+# unpack
+tar -xvzf stine_Darwin_x86_64.tar.gz
+# add to $PATH
+sudo cp ./stine ~/../../usr/local/bin
```
Comment (irgendwr) [inline README.md]: the relative path doesn't make sense to me
```suggestion
sudo cp ./stine /usr/local/bin
```

File: README.md [hunk 2]
```diff
@@ -33,6 +33,21 @@ sudo cp ./stine /usr/bin/stine
 Create a file called `.stine.yaml` (either inside your home folder or the folder containing the program) (e.g. using `nano ~/.stine.yaml`) and edit it to fit your needs.
 See [config](#config) section for examples.
 
+### macOS
+Download and unpack the latest release:
+```bash
```
Comment (irgendwr) [inline README.md]: please add an empty line between the heading, text and code block
(_I know that I also broke the markdown rules - do as I say, not as I do_ :sweat_smile: )

**Comment by irgendwr** [comment]
Also: feel free to open another PR that adds your library in the "Related Projects" section of the [readme](https://github.com/irgendwr/go-stine?tab=readme-ov-file#related-projects).

File: README.md (hunk 1 above)
Comment (martenmatrix) [reply README.md]: yes, that looks more precise, i've just copied the commands i've executed in my home directory :)

File: README.md (hunk 2 above)
Comment (martenmatrix) [reply README.md]: should have checked the preview 😅

**Comment by irgendwr** [comment]
thx! :)

**Pull request merged**